OpenSSL-Query/README.md

```diff
@@ -58,12 +57,14 @@ Testing
 
 Testing is done like this:
 
-    $ make test
+    make test
 
-However, it requires that a temporary query service is started as
-well.  This is part of QueryApp, and is started like this:
+However, it requires the additional Perl module Dancer2 and
+plackup (debian packages libdancer2-perl and libplack-perl)
+and that a temporary query service is started as well.
+This is part of QueryApp, and is started like this:
 
-    $ here=`pwd`
-    $ cd ../QueryApp	# Or wherever you have it checked out
-    $ PERSONDB=./t/query_data/pdb.yaml CLADB=./t/query_data/cdb.txt \
-      PERL5LIB=./lib:$here/lib plackup bin/query.psgi
+    here=`pwd`        # the directory OpenSSL-Query/
+    cd ../QueryApp	# Or wherever you have it checked out
```

Comment:
Nit: align the comments, and most of all, I prefer to see a little more distance between the command and the comment, for readability

Reply:
Ah, there was a nasty TAB character in between - removed.
I've also put more space as you suggested.